modular/gater/object_handler.go

```diff
@@ -538,7 +538,6 @@ func (g *GateModular) getObjectHandler(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 	w.Header().Set(ContentTypeHeader, objectInfo.GetContentType())
-	w.Header().Set(ContentDispositionHeader, ContentDispositionAttachmentValue+"; filename=\""+reqCtx.objectName+"\"")
```

Comment:
Are we just removing the content-disposition assginment and not set a switch for view/download?

Reply:
we did already. 
The code being removed is redundant.